docs/userguide/main.md

```diff
@@ -910,3 +910,73 @@ spec:
     opensearchCredentialsSecret:
       name: dashboards-credentials  # This is the name of your secret that contains the credentials for Dashboards to use
 ```
+
+### Configuring Snapshot Repo (BETA):
```

Comment:
This is at the wrong place in the userguide now (there was a restructuring some weeks ago). Please move it under the `Configuring OpenSearch` section.

Reply:
Updated the location.